plugins/progress.go

```
@@ -57,41 +66,78 @@ const (
 func (plugin ProgressPlugin) Check(offset interface{}, context PluginContext) (interface{}, error) {
 	context.Info.Println("Checking for progress updates...")
 
+	// Parse debounce delay
```

No need to perform parsing here (and if it failed, we should return the error, anyway). Instead the struct can directly use a `time.Duration` value. The parser is aware of how to parse time values.

---

Good to know. I'll update it in the next revision.

---

Updated in the latest revision.